challenge_1/python/ekstrah/src/a.py

```diff
@@ -0,0 +1,5 @@
+a = input("Type anything you want to print backwards : ")
+
+for i in range(len(a)-1, -1, -1): #{
+    print(a[i], end="")
+#}
```

Comment:
This works but you'll find the more common solution is to do a string slice.
Take a look through [this stuff](https://www.digitalocean.com/community/tutorials/how-to-index-and-slice-strings-in-python-3) and see if you can find the one i'm talking about.